pkgs/applications/networking/mailreaders/mmh/default.nix

```
@@ -10,6 +10,10 @@ in stdenv.mkDerivation rec {
     sha256 = "1q97p4g3f1q2m567i2dbx7mm7ixw3g91ww2rymwj42cxk9iyizhv";
   };
 
+  postPatch = stdenv.lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) ''
```

```suggestion
  postPatch = ''
```
When using "$(AR)", can make the change unconditional.